### PR TITLE
Create maintainer_guide.fre.md

### PR DESCRIPTION
**Pull request by kushal34712**
converted plz accpet it

### CODE REVIEW

**Comment by kushal34712** [comment]
@ascodeasice what more chanage i need to do ?

**Comment by kushal34712** [comment]
sir just accpet it then as per preview and all i will make changes in it

File: translations/maintainer_guide/maintainer_guide.french.md
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.french.md:1]: file naming is incorrect

File: translations/maintainer_guide/maintainer_guide.french.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Si jamais vous avez des doutes, vous pouvez toujours me mentionner ou mentionner les autres responsables dans le PR lui-même ou m'envoyer un message privé sur [Twitter](https://twitter.com/Syknapse).
+
+## Rejoignez-nous
+
+Rejoignez-nous pour contribuer à faire grandir ce projet ensemble. Contactez-moi au [Twitter](https://twitter.com/Syknapse) et envoyez-moi votre nom d'utilisateur GitHub pour que je puisse vous ajouter. Vous pouvez également rejoindre notre communauté Discord en cliquant sur le bouton ci-dessous :
+
+[![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Join%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Join our Discord server!')
+
+##Merci
+
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.french.md:74]: what is this for?

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@kushal34712, kindly make the requested changes. Thanks

**Comment by kushal34712** [comment]
@TenTraicion ##Merci = thank you

**Comment by kushal34712** [comment]
i have made all the changes check it out

File: translations/maintainer_guide/maintainer_guide.french.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Traductions
+
+Ce tutoriel est également disponible en [autres langues](translations/README.md)
+
+| [English](maintainer_guide.md)   | [Bangla](translations/maintainer_guide/maintainer_guide.ben.md) | [Chinese (Traditional)](/translations/maintainer_guide/maintainer_guide.zho-tc.md) | [German](translations/maintainer_guide/maintainer_guide.ger.md) | [Hindi](translations/maintainer_guide/maintainer_guide.hin.md) |
+| :---: | :---: | :---: | :---: | :---: |
+| [Italian](translations/maintainer_guide/maintainer_guide.ita.md) | [Japanese](translations/maintainer_guide/maintainer_guide.jpn.md) | [Portuguese](translations/maintainer_guide/maintainer_guide.por.md) | [Ukrainian](/translations/maintainer_guide/maintainer_guide.ukr.md) |
+
+
+> Les traductions des documentations de projets sont les bienvenues. Lire [`Guide de traduction`](translations/README.md)contribuer.
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.french.md:14]: broken links

File: translations/maintainer_guide/maintainer_guide.french.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Cependant, il n'est pas rare d'avoir quelques PR en attente et c'est alors qu'il y aura inévitablement des conflits de fusion. Vous pouvez utiliser tous les outils que vous connaissez pour voir les différences et résoudre les conflits.
+I recommend the use of a tool like [GitKraken](https://www.gitkraken.com/download). C'est visuel et cela permet une gestion plus facile du projet lorsqu'il y a quelques PR à réaliser.
+Téléchargez GitKraken, clonez le projet. L'utilisation d'une combinaison de votre éditeur de code et de l'outil de conflit de fusion intégré de GitKraken vous donne un contrôle total pour parcourir rapidement les PR, résoudre les conflits et fusionner.
+
+Le projet a installé Prettier pour garantir que quelle que soit la manière dont un contributeur soumet le PR, le guide de style sera appliqué. Le projet de cette façon est toujours maintenu avec la même indentation et le même style.
+Si vous remarquez que le fichier HTML semble désordonné, exécutez le code ci-dessous à la racine du projet.
+
+```js
+npx plus joli --write index.html
+```
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.french.md:61]: don't translate the codes

File: translations/maintainer_guide/maintainer_guide.french.md
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.french.md:1]: file name should be `maintainer_guide.fra.md`
kindly change the file name.

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@kushal34712, still need some changes mentioned below:

**Comment by TenTraicion** [comment]
you may follow other translations to fix the links. Thanks

**Comment by kushal34712** [comment]
i have made update con u check

**Comment by kushal34712** [comment]
@TenTraicion all links are working just check it once

File: translations/maintainer_guide/maintainer_guide.fra.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+Si jamais vous avez des doutes, vous pouvez toujours me mentionner ou mentionner les autres responsables dans le PR lui-même ou m'envoyer un message privé sur [Twitter](https://twitter.com/Syknapse).
+
+## Rejoignez-nous
+
+Rejoignez-nous pour contribuer à faire grandir ce projet ensemble. Contactez-moi au [Twitter](https://twitter.com/Syknapse) et envoyez-moi votre nom d'utilisateur GitHub pour que je puisse vous ajouter. Vous pouvez également rejoindre notre communauté Discord en cliquant sur le bouton ci-dessous :
+
+[![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Join%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Join our Discord server!')
+
+
+
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.fra.md:73]:
```suggestion
[![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Join%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Join our Discord server!')
```

File: translations/maintainer_guide/maintainer_guide.fra.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+S'il n'y a pas beaucoup de PR accumulés, tout ce processus peut être effectué directement dans la page GitHub du projet.
+Cependant, il n'est pas rare d'avoir quelques PR en attente et c'est alors qu'il y aura inévitablement des conflits de fusion. Vous pouvez utiliser tous les outils que vous connaissez pour voir les différences et résoudre les conflits.
+I recommend the use of a tool like [GitKraken](https://www.gitkraken.com/download). C'est visuel et cela permet une gestion plus facile du projet lorsqu'il y a quelques PR à réaliser.
+Téléchargez GitKraken, clonez le projet. L'utilisation d'une combinaison de votre éditeur de code et de l'outil de conflit de fusion intégré de GitKraken vous donne un contrôle total pour parcourir rapidement les PR, résoudre les conflits et fusionner.
+
+Le projet a installé Prettier pour garantir que quelle que soit la manière dont un contributeur soumet le PR, le guide de style sera appliqué. Le projet de cette façon est toujours maintenu avec la même indentation et le même style.
+Si vous remarquez que le fichier HTML semble désordonné, exécutez le code ci-dessous à la racine du projet.
+
+```js
+npx plus joli --write index.html
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.fra.md:59]:
```suggestion
npx prettier --write index.html
```

File: translations/maintainer_guide/maintainer_guide.fra.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+Vous pouvez y consacrer autant ou aussi peu de temps que vous le souhaitez. Entre nous, nous espérons que tout se passera bien.
+
+## Méthodologie
+
+- Accédez à la section pull request du projet, commencez par la pull request la plus ancienne qui n'est pas dans l'état « modifications demandées ».
+- Ouvrez un PR et accédez à l'onglet Modifications des fichiers et lancez une révision du code.
+- Vérifiez le PR, assurez-vous qu'il suit les spécifications du didacticiel.
+- Assurez-vous que le HTML, les liens et les données sont tous corrects. Assurez-vous que la carte est positionnée au début du fichier là où elle devrait être.
+- Vérifiez ensuite tout conflit. Fusionnez « master » dans la branche PR pour résoudre les conflits. Les conflits surviennent généralement lorsqu'un certain temps s'est écoulé depuis les fusions précédentes et que plusieurs PR utilisent la même version obsolète.
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.fra.md:34]:
```suggestion
- Vérifiez ensuite tout conflit. Fusionnez `master` dans la branche PR pour résoudre les conflits. Les conflits surviennent généralement lorsqu'un certain temps s'est écoulé depuis les fusions précédentes et que plusieurs PR utilisent la même version obsolète.
```

File: translations/maintainer_guide/maintainer_guide.fra.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+| :---: | :---: | :---: | :---: | :---: |
+| [Italian]([/translations/maintainer_guide/maintainer_guide.ita.md](https://github.com/Syknapse/Contribute-To-This-Project/blob/69f692fc596e46dfe624e3d263b458defd31b258/translations/maintainer_guide/maintainer_guide.ita.md) | [Japanese](https://github.com/Syknapse/Contribute-To-This-Project/blob/69f692fc596e46dfe624e3d263b458defd31b258/translations/maintainer_guide/maintainer_guide.jpn.md) | [Portuguese](https://github.com/Syknapse/Contribute-To-This-Project/blob/69f692fc596e46dfe624e3d263b458defd31b258/translations/maintainer_guide/maintainer_guide.por.md) | [Ukrainian](https://github.com/Syknapse/Contribute-To-This-Project/blob/69f692fc596e46dfe624e3d263b458defd31b258/translations/maintainer_guide/maintainer_guide.ukr.md) |
+
+> Les traductions des documentations de projets sont les bienvenues. Lire [`Guide de traduction`](https://github.com/Syknapse/Contribute-To-This-Project/blob/69f692fc596e46dfe624e3d263b458defd31b258/translations/README.md)contribuer.
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.fra.md:13]:
```suggestion
> Les traductions des documentations de projets sont les bienvenues. Lire [`Guide de traduction`](../README.md) contribuer.
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]

**Comment by kushal34712** [comment]
@TenTraicion have u checked

**Comment by kushal34712** [comment]
Sir the changes can u re open it and mere it bcz it is not reflecting in my account

**Pull request merged**